### PR TITLE
List All Wishlists

### PR DESCRIPTION
**Pull request by rahilkadakia**
#7 

1. Added endpoint for `list_wishlists`.
2. Added test route `test_get_wishlist_list`.
3. Code Coverage: 97%.
4. Lint Score 10/10

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -148,3 +148,14 @@ def test_method_not_allowed(self):
...
+    def test_get_wishlist_list(self):
+        """It should Get a list of Wishlists"""
+        self._create_wishlists(5)
+        resp = self.client.get(
+            BASE_URL,
+            content_type="application/json"
+        )
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        data = resp.get_json()
+        self.assertEqual(len(data), 5)
```
Comment (studentgrad0419) [inline tests/test_routes.py:161]: One thing that can also be tested is the contents if the contents of the returned list matches the ones you've created.
Other than that, it looks fine.

**Review by studentgrad0419 — APPROVED** [review]
Hi Rahil, everything looks fine. I made a comment/ suggestion to make it possibly more robust, but works for the story in this sprint.

File: tests/test_routes.py (hunk 1 above)
Comment (rahilkadakia) [reply tests/test_routes.py]: @studentgrad0419 added recommended changes, thanks

**Review by studentgrad0419 — APPROVED** [review]
Looks great!

**Pull request merged**